npm run lint may not output error messages when running errors

```bash
 ✘ binsee@shanmudeMBP15 > ~/CodeWork/puppet-xp > > fix-git-scripts ± > git push                   
[Step-1] Checking last commit... 

[Step-2] Checking lint... 

Error:
------------------------------------------



------------------------------------------

Failed to exec pre-push hook script
error: 无法推送一些引用到 'github.com:binsee/puppet-xp.git'
```

Running npm lint fails with no error message


- check

```bash
 ✘ binsee@shanmudeMBP15 > ~/CodeWork/puppet-xp > > fix-git-scripts ± > npm run lint                     

> wechaty-puppet-xp@1.10.26 lint
> npm run lint:es && npm run lint:ts && npm run lint:md


> wechaty-puppet-xp@1.10.26 lint:es
> eslint "src/**/*.ts" "tests/**/*.spec.ts" --ignore-pattern tests/fixtures/

=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.5.0

YOUR TYPESCRIPT VERSION: 4.5.5

Please only submit bug reports when using the officially supported version.

=============

> wechaty-puppet-xp@1.10.26 lint:ts
> tsc --isolatedModules --noEmit

examples/ding-dong-bot.ts:113:5 - error TS2339: Property 'talkerId' does not exist on type 'MessagePayload'.

113     talkerId,
        ~~~~~~~~

src/puppet-xp.ts:307:7 - error TS2322: Type '{ id: string; roomId: string; talkerId: string; text: string; timestamp: number; toId: string; type: PUPPET.type.Message.Unknown | PUPPET.type.Message.Audio | PUPPET.type.Message.Contact | ... 4 more ... | PUPPET.type.Message.Video; }' is not assignable to type 'MessagePayload'.
  Object literal may only specify known properties, and 'talkerId' does not exist in type 'MessagePayload'.

307       talkerId,
          ~~~~~~~~


Found 2 errors.
```


```bash
 ✘ binsee@shanmudeMBP15 > ~/CodeWork/puppet-xp > > fix-git-scripts ± > npm run lint:ts            

> wechaty-puppet-xp@1.10.26 lint:ts
> tsc --isolatedModules --noEmit

examples/ding-dong-bot.ts:113:5 - error TS2339: Property 'talkerId' does not exist on type 'MessagePayload'.

113     talkerId,
        ~~~~~~~~

src/puppet-xp.ts:307:7 - error TS2322: Type '{ id: string; roomId: string; talkerId: string; text: string; timestamp: number; toId: string; type: PUPPET.type.Message.Unknown | PUPPET.type.Message.Audio | PUPPET.type.Message.Contact | ... 4 more ... | PUPPET.type.Message.Video; }' is not assignable to type 'MessagePayload'.
  Object literal may only specify known properties, and 'talkerId' does not exist in type 'MessagePayload'.

307       talkerId,
          ~~~~~~~~


Found 2 errors.
```

- turn off display stdout

```bash
 ✘ binsee@shanmudeMBP15 > ~/CodeWork/puppet-xp > > fix-git-scripts ± > npm run lint:ts > /dev/null
```

So, error message for `tsc --isolatedModules --noEmit` not output to stderr, but output to stdout.

We need to be compatible with the message output to stdout when stderr has no message.

After pr #30 

```bash
✘ binsee@shanmudeMBP15  ~/CodeWork/puppet-xp   fix-git-scripts ±  git push                   
[Step-1] Checking last commit... 

[Step-2] Checking lint... 

Error:
------------------------------------------


> wechaty-puppet-xp@1.10.26 lint
> npm run lint:es && npm run lint:ts && npm run lint:md


> wechaty-puppet-xp@1.10.26 lint:es
> eslint "src/**/*.ts" "tests/**/*.spec.ts" --ignore-pattern tests/fixtures/


> wechaty-puppet-xp@1.10.26 lint:ts
> tsc --isolatedModules --noEmit

examples/ding-dong-bot.ts(113,5): error TS2339: Property 'talkerId' does not exist on type 'MessagePayload'.
src/puppet-xp.ts(307,7): error TS2322: Type '{ id: string; roomId: string; talkerId: string; text: string; timestamp: number; toId: string; type: PUPPET.type.Message.Unknown | PUPPET.type.Message.Audio | PUPPET.type.Message.Contact | ... 4 more ... | PUPPET.type.Message.Video; }' is not assignable to type 'MessagePayload'.
  Object literal may only specify known properties, and 'talkerId' does not exist in type 'MessagePayload'.


------------------------------------------

Failed to exec pre-push hook script
error: 无法推送一些引用到 'github.com:binsee/puppet-xp.git'
```

--- a/bin/pre-push.ts
+++ b/bin/pre-push.ts
@@ -109,7 +109,7 @@ process.exit(42)
 function checkReturn (ret: ShellString) {
   if (ret.code !== 0) {
     const line = '------------------------------------------'
-    console.error(`Error:\n${line}\n\n${ret.stderr}\n\n${line}\n`)
+    console.error(`Error:\n${line}\n\n${ret.stderr || ret.stdout}\n\n${line}\n`)
     process.exit(1)
   }
   return ret

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@chatie/git-scripts",
-  "version": "0.7.9",
+  "version": "0.7.10",
   "description": "Git Hooks Integration for Chatie Projects",
   "directories": {
     "doc": "docs",

